### PR TITLE
Fix list in SSA docs

### PR DESCRIPTION
**Pull request by apelisse**
That's it

### CODE REVIEW

**Comment by apelisse** [comment]
That fixes the bug!

File: content/en/docs/reference/using-api/api-concepts.md [hunk 1]
```diff
@@ -543,10 +543,11 @@ used when merging objects of this type.
...
 simplify the update logic of your controller. The main differences with a
 read-modify-write and/or patch are the following:
-- the applied object must contain all the fields that the controller cares about.
-- there are no way to remove fields that haven't been applied by the controller
+
+* the applied object must contain all the fields that the controller cares about.
+* there are no way to remove fields that haven't been applied by the controller
   before (controller can still send a PATCH/UPDATE for these use-cases).
-- the objet doesn't have to be read beforehand, `resourceVersion` doesn't have
+* the objet doesn't have to be read beforehand, `resourceVersion` doesn't have
```
Comment (liggitt) [inline content/en/docs/reference/using-api/api-concepts.md]: while you're here, s/objet/object/

Comment (apelisse) [reply content/en/docs/reference/using-api/api-concepts.md]: French spelling ... :-)

**Pull request merged**